test/testnet/localnet/Makefile

```diff
@@ -0,0 +1,47 @@
+.PHONY: deploy stop update-prereq build-tendermint status
+
+ACTIVATE_VENV := source venv/bin/activate
+TESTNET ?= tik
+
```

Comment:
`GOPATH ?= $(shell go env GOPATH)` is necessary for line 8 on systems that didn't set GOPATH explicitly.